### DISCUSSION TITLE
Does the heltec wireless tracker have secondary s2i bus?

### DISCUSSION THREAD

**Discussion by Nikguy321** [meshtastic/firmware#2705] [Q&A] [answered]
I want to connect a card KB to the heltec wireless teacker. Does it have a second sda/scl pin for canned messaging?   If not, is there a good wiring diagram to show how I can connect rhe keyboard for canned messages? Thanks.

**Comment by HarukiToreda** [comment, marked as answer]
45(SDA) and 46(SCL) are the I2C pins on the firmware.
